src/main/groovy/com/bmuschko/gradle/docker/internal/RegistryAuthLocator.groovy

```diff
@@ -106,6 +106,9 @@ class RegistryAuthLocator {
             configFile.exists() ? 'exists' : 'does not exist',
             commandPathPrefix)
 
+        if (!(configFile.exists())) {
```

Comment:
I am OK with adding this in, however, it will only cover some of the issues users reported.